### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Dorian Kury

### DISCUSSION THREAD

**Discussion by nairod22** [heig-vd-dai-course/heig-vd-dai-course#289] [Show and tell]
The link to your GitHub profile README!

**Comment by ludelafo** [comment]
It's very interesting the transition you made from physical therapist! Do you still practice from time to time? How do you find yourself in computer sciences now?

**Reply by nairod22** [reply]
Yes, I keep practicing part time while studying. I am really excited with the future of healthcare. Many innovative solutions will flood over the market in coming years helping practitioners. Also happy to evolve into a more rational and technical field.

**Reply by ludelafo** [reply]
That's really cool, thank you for sharing!
